Makefile

```diff
@@ -14,9 +14,17 @@ DOC_DIR := $(PROJECT_DIR)/doc
 DOC := $(DOC_DIR)/swagger.json
 
 GO_ENV := GOPATH=$(GOPATH)
+GO_LINT := PATH=${PATH}:$(GOPATH)/bin golint
```

Comment:
I'm not sure we want to introduce GO_LINT at this point - I had problems with it. Is our current codebase passing lint?